### DISCUSSION TITLE
SSO login

### DISCUSSION THREAD

**Discussion by phxgbl** [directus/directus#6650] [Feature Requests]
Simple question. Can we use Keycloak authorization code flow (SSO) for Directus login. If it's not possible out of box.. Is there any way to accomplish that. I mean code change Or other ways?

**Comment by rijkvanzanten** [comment]
Afaik keycloak supports the oAuth flow, which makes it compatible with Directus. Directus relies on `grant` for it's oAuth flow, which can be configured through the `OAUTH_` environment variables (https://docs.directus.io/reference/environment-variables/#oauth)

**Reply by phxgbl** [reply]
@rijkvanzanten   that's what I wanted, grant_type = authorization_code.I will give it a try. Thank you.
